rdkit/Chem/EState/UnitTestVSA.py

```diff
@@ -12,52 +12,44 @@
 
 """
 from __future__ import print_function
-from rdkit import RDConfig
-import unittest, os
+
+import os
+import unittest
+
 from rdkit import Chem
 from rdkit.Chem.EState import EState_VSA
-import os.path
 
 
-def feq(n1, n2, tol=1e-4):
-  return abs(n1 - n2) <= tol
+class TestCase(unittest.TestCase):
 
+  @staticmethod
```

Comment:
I don't understand at all why you made this change.
It doesn't seem like a simplification or clarification.
What's the motivation?